test/mysql_handler/test

```diff
@@ -1,20 +1,55 @@
 #!/bin/bash -ex
 
 local_test=false
-while getopts :l opt; do
+while getopts ":lb" opt; do
     case $opt in
         l) local_test=true ;;
+				b) benchmark=true ;;
```

Comment:
fix indentation

Reply:
I don't know why it looks funky on here but on my end, it is all aligned. Will untab and tab again to make sure